themes/src/main/resources/theme/base/admin/resources/partials/client-service-account-roles.html

```diff
@@ -7,10 +7,23 @@
 
     <kc-tabs-client></kc-tabs-client>
 
-    <h2><span>{{client.clientId}}</span> {{:: 'service-accounts' | translate}} </h2>
-    <p class="subtitle"></p>
-
     <form class="form-horizontal" name="realmForm" novalidate kc-read-only="!client.access.configure" data-ng-show="client.serviceAccountsEnabled">
+
+        <h2>{{:: 'service-account' | translate}}</h2>
+
+        <div class="form-group">
+            <label class="col-md-2 control-label">{{:: 'user-name' | translate}}</label>
+            <div class="col-md-6">
+                <a class="form-control" ng-href="#/realms/{{realm.realm}}/users/{{user.id}}" target="_blank">{{user.username}}</a>
```

Comment:
I understand why you're doing _blank here, but I would just open it directly. I can always go back with back button. 
Should just render as a link rather than as an input field.

Reply:
Good point. I just removed the target=_blank